source/collaborate/schedule-messages.rst

```diff
@@ -0,0 +1,43 @@
+Schedule messages
+==================
+
+.. include:: ../_static/badges/ent-pro-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+There are times when you want to send a message, but you don't want it sent immediately. For example, it may be after working hours for the recipient. 
+From Mattermost v10.3, using Mattermost in a web browser or the desktop app, you can schedule messages to send in the future. 
+
+Compose a message, then select the right side of the **Send** icon |send-icon| to schedule when the message will be sent. You can choose from a fixed time or set a custom time.
+
+.. image:: ../images/scheduled-messages.gif
+    :alt: Compose a message and select the right side of the Send icon to schedule when the message will be sent.
+
+.. tip::
+
+    When scheduling a direct message, Mattermost displays both your local time and the recipient's local time.
+
+Manage scheduled messages
+-------------------------
+
+Once you schedule a message to send in the future, that message is available within the **Drafts** view, under the **Scheduled** tab.
+
+You can manage scheduled messages with the following actions:
+
+- **Delete scheduled post**: Confirm that you want to delete the message.
+- **Edit scheduled post**: Make changes to the draft inline before its sent.
+- **Copy text**: Copy the draft text.
+- **Reschedule message**: Change when the message should be sent.
+- **Send now**: Confirm that you want to send the message immediately.
+
+.. image:: ../images/manage-scheduled-message.png
+    :alt: Once a message is scheduled, you can delete, edit, and reschedule it, copy the message text, or send it immediately.
+
+Troubleshoot scheduled messages
+-------------------------------
+
+If a scheduled message fails to send, you'll be alerted with a banner at the top of the Mattermost screen. Scheduled messages may fail for the following reasons:
```

Comment:
> you'll be alerted with a banner at the top of the Mattermost screen

The alert banner is only visible in the scheduled tab (same place where you see the scheduled posts). 

Additionally, you also get a red badge for scheduled post count in the LHS if there are some scheduled posts with error. Note that the count of scheduled even in the red badge is a total, and not how many scheduled posts have error. Stating this as this can be a point of confusion.

![Screenshot 2024-12-03 at 9 51 34 AM](https://github.com/user-attachments/assets/7e0138a4-39f2-4c2c-8f57-debaf9cdf7e2)

Reply:
@matthewbirtch - I didn't see the new badge in the [compass icons](https://github.com/mattermost/compass-icons) repository (though I may have missed it). Can you point me to these new icons, please?

Reply:
@cwarnermm The badge itself is not an icon that would be in compass icons. If you would like a PNG image here's one you can use:
![image](https://github.com/user-attachments/assets/a4a201c2-a102-4188-af2d-52d7d5478106)